### ISSUE TITLE
Suggestion: Recreate original gamelog.txt output

### ISSUE THREAD

**Issue by themaster567** [DFHack/dfhack#2579]
There are several utilities, Soundsense obviously being most notable, that relied entirely upon getting information from gamelog.txt. The problem is that the Steam release has vastly neutered what's actually sent to that file now. I am not an expert on what is possible with DFHack, but I am guessing (and perhaps hoping) that this is something that it could do.

**Comment by lethosor** [comment]
Yes, we have `devel/annc-monitor`, which prints all announcements/reports to the console, and can be trivially modified to print to a file instead / as well.

**Comment by TheBloke** [comment]
This would be excellent. I like the logs going to the console, but when using the console for other DFHack stuff it gets a bit busy. Having the output in a file again that can be tailled would be great.

Ideally I would extend the idea even further: a new gui window that can display the log output from annc-monitor, being brought up on demand at any time to view and scroll through the logs, or resized and positioned and locked in place for permanent display. That would be much more accessible and convenient for many users.
